govcd/cse/4.2.0/rde.tmpl

```diff
@@ -0,0 +1,32 @@
+{
+  "apiVersion": "capvcd.vmware.com/v1.1",
+  "kind": "CAPVCDCluster",
+  "name": "{{.Name}}",
+  "metadata": {
+    "name": "{{.Name}}",
+    "orgName": "{{.Org}}",
+    "site": "{{.VcdUrl}}",
+    "virtualDataCenterName": "{{.Vdc}}"
+  },
+  "spec": {
+    "vcdKe": {
+      "isVCDKECluster": true,
+      "markForDelete": {{.Delete}},
+      "forceDelete": {{.ForceDelete}},
+      "autoRepairOnErrors": {{.AutoRepairOnErrors}},
+      {{- if .DefaultStorageClassName }}
+      "defaultStorageClassOptions": {
+          "filesystem": "{{.DefaultStorageClassFileSystem}}",
+          "k8sStorageClassName": "{{.DefaultStorageClassName}}",
+          "vcdStorageProfileName": "{{.DefaultStorageClassStorageProfile}}",
+          "useDeleteReclaimPolicy": {{.DefaultStorageClassUseDeleteReclaimPolicy}}
+      },
+      {{- end }}
+      "secure": {
+        "apiToken": "{{.ApiToken}}"
```

Comment:
If rde template has spec.vcdKe.secure.apiToken, remove secret object from the capiYaml template

Reply:
Good catch, I will.